bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tfile.py

```diff
@@ -15,12 +15,71 @@
 
 from libROOTPython import AddFileOpenPyz
 from ROOT import pythonization
+import sys, errno, os
+from functools import partial
+
+# We check the Python version because FileNotFoundError
+# is not available in Python2
+def RaiseError(file_name):
```

Comment:
The name seems to be confusing as it seems to imply this routine 'always' throw an exception.